### PR TITLE
Message against the war, in Russian and English

### PR DESCRIPTION
**Pull request by afilina**
As per the request of the technical steering committee, here is the message regarding the war, in two languages. Related issue: #111

### CODE REVIEW

File: .github/to-russians.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## 🇷🇺 Русским гражданам
+
+Мы, участники Laminas, живем в разных странах. У многих из нас есть друзья, родственники и коллеги как в России, так и в Украине. Некоторые из нас родились в России. Некоторые из нас живут в России. У некоторых есть бабушки и дедушки, которые сражались с фашистами во Второй мировой войне. Здесь никто не поддерживает фашизм.
+
+У одного из нас есть украинская родственница, которая спаслась из дома вместе с сыном. Поезд задержался из-за бомбежки на дороге впереди. У нас есть друзья, которые прячутся в бомбоубежищах. Мы с тревогой ждем весточки от них после воздушных налетов, которые беспорядочно наносят удары и попадают по больницам, школам, детским садам и домам. Мы не берем это из каких-либо СМИ. Это наши собственные наблюдения.
+
+Вы доверяете нам достаточно, чтобы использовать наше программное обеспечениеи, и мы просим вас довериться нам вновь. Мы нуждаемся в помощи. Выходите и протестуйте против этой бесполезной войны. Остановите кровопролитие. Скажите "Нет войне!"
```
Comment (Xerkus) [inline .github/to-russians.md]: "программное обеспечение" is a bit too formal
```suggestion
Вы доверяете нам достаточно, чтоб использовать наши программы, и мы просим вас довериться нам вновь. Мы нуждаемся в помощи. Выходите и протестуйте против этой бесполезной войны. Остановите кровопролитие. Скажите "Нет войне!"
```

File: .github/to-russians.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+## 🇷🇺 Русским гражданам
+
+Мы, участники Laminas, живем в разных странах. У многих из нас есть друзья, родственники и коллеги как в России, так и в Украине. Некоторые из нас родились в России. Некоторые из нас живут в России. У некоторых есть бабушки и дедушки, которые сражались с фашистами во Второй мировой войне. Здесь никто не поддерживает фашизм.
+
+У одного из нас есть украинская родственница, которая спаслась из дома вместе с сыном. Поезд задержался из-за бомбежки на дороге впереди. У нас есть друзья, которые прячутся в бомбоубежищах. Мы с тревогой ждем весточки от них после воздушных налетов, которые беспорядочно наносят удары и попадают по больницам, школам, детским садам и домам. Мы не берем это из каких-либо СМИ. Это наши собственные наблюдения.
```
Comment (afilina) [inline .github/to-russians.md]:
```suggestion
У одного из нас есть украинская родственница, которая спаслась из дома вместе с сыном. Поезд задержался из-за бомбежки на дороге впереди. У нас есть друзья, которые прячутся в бомбоубежищах. Мы с тревогой ждем весточки от них после воздушных налетов, которые беспорядочно наносят удары и попадают по больницам, школам, детским садам и домам. Мы не берем это из каких-либо СМИ. Мы наблюдаем это напрямую.
```

File: .github/to-russians.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+## 🇷🇺 Русским гражданам
+
+Мы, участники Laminas, живем в разных странах. У многих из нас есть друзья, родственники и коллеги как в России, так и в Украине. Некоторые из нас родились в России. Некоторые из нас живут в России. У некоторых есть бабушки и дедушки, которые сражались с фашистами во Второй мировой войне. Здесь никто не поддерживает фашизм.
```
Comment (Xerkus) [inline .github/to-russians.md]:
```suggestion
Мы, участники Laminas, родились и живем в разных странах. У многих из нас есть друзья, родственники и коллеги как в России, так и в Украине. Некоторые из нас родились в России. Некоторые из нас живут в России. У некоторых бабушки и дедушки сражались с фашистами во Второй мировой войне. Здесь никто не поддерживает фашизм.
```

**Review by Xerkus — COMMENTED** [review]
Took a stab at slightly improving translation.

**Review by Xerkus — APPROVED** [review]

**Review by Ocramius — APPROVED** [review]
LGTM - I need some help in getting this into our docs somehow (ideally everywhere).

**Comment by Ocramius** [comment]
Note: this is being added to `README.md` across the whole organization. See https://laminas.slack.com/archives/C011F9W87E1/p1646746797320559

**Comment by kynx** [comment]
Thank you so much for writing this @afilina. It’s simple and powerful and direct. 

This is trivial in the context, but yesterday my bosses asked me to identify any “Russian” software we use. My mouth just hung open.

I am hoping your words - and those of this community and all the others whose work pays my bosses’ mortgages - can change their way of thinking about the world. I’m an optimist I know. But I have to try and this will help.

**Comment by boesing** [comment]
Closing here as this is already part of the `README.md`, right?
If not, we can re-open this ofc.

**Pull request closed without merging**